meson.build

```diff
@@ -36,8 +36,8 @@ headers = [include_directories('inc'), include_directories('.')]
 subdir('inc')
 subdir('src')
 
-if get_option('example') == true
-    subdir('example')
+if get_option('util') == true
```

Comment:
util => utility, 
This is not a mandatory feature, so please default is false

Reply:
I renamed the option name and moved it to src/bin.